src/components/pages/Site/Site.js

```diff
@@ -114,6 +115,11 @@ const SiteForm = ({
   handleLatitudeChange,
   handleLongitudeChange,
 }) => {
+  const handleLngLatSwap = () => {
+    handleLatitudeChange(formik.getFieldProps('longitude').value)
+    handleLongitudeChange(formik.getFieldProps('latitude').value)
```

Comment:
should we store one or both of these in an intermediate value in case latitude gets updated so fast on line 120 when the code reads the value from latitude, its the new longitude value?

Reply:
I'm a bit confused - `formik.getFieldProps('longitude').value` is the most recent value. We're use similarly on `ProjectInfo.js` and `ManagemenetRegime.js`